lib/paramedic.js

```diff
@@ -377,42 +381,37 @@ class ParamedicRunner {
     }
 
     getCommandForStartingTests () {
-        let cmd = this.config.getCli() + ' ' + this.config.getAction() + ' ' + this.config.getPlatformId() + utilities.PARAMEDIC_COMMON_CLI_ARGS;
-
-        const addConfigArgs = (cmd) => {
-            if (this.config.getArgs()) {
-                cmd += ' ' + this.config.getArgs();
-            }
-
-            return cmd;
-        };
-
-        if (this.config.getPlatformId() === utilities.BROWSER) {
-            return addConfigArgs(cmd);
-        }
-
-        const paramedicTargetChooser = new ParamedicTargetChooser(this.tempFolder.name, this.config);
-
-        if (this.config.getAction() === 'build' || (this.config.getPlatformId() === utilities.WINDOWS && this.config.getArgs().indexOf('appx=8.1-phone') < 0)) {
+        let cmd = [
+            this.config.getCli(),
+            this.config.getAction(),
+            this.config.getPlatformId()
+        ]
+            .concat(utilities.PARAMEDIC_COMMON_ARGS)
```

Comment:
this formatting is weird, but I gues correct?

Reply:
Yeah, ESLint complains if it was not indented.